src/main/webapp/view/discussion/discussion-area.js

```diff
@@ -27,6 +26,10 @@ export const ELEMENT_DISCUSSION =
 export default class DiscussionArea {
   static #ELEMENT_POST_TEXTAREA = document.querySelector('#post-textarea');
   static #ELEMENT_TIMESTAMP_SPAN = document.querySelector('#timestamp-span');
+  static #ELEMENT_NEW_COMMENT_TYPES =
+      document.querySelector('#new-comment-types');
+  static #SELECTOR_SELECTED_TYPE = 'label.active > input';
```

Comment:
This is confusing can you add a comment here

Reply:
added one